addons/event/security/ir.model.access.csv

```diff
@@ -26,3 +26,4 @@ access_event_tag,event.tag,model_event_tag,,0,0,0,0
 access_event_tag_registration,event.tag.user,model_event_tag,event.group_event_registration_desk,1,0,0,0
 access_event_tag_user,event.tag.user,model_event_tag,event.group_event_user,1,1,1,0
 access_event_tag_manager,event.tag.manager,model_event_tag,event.group_event_manager,1,1,1,1
+access_ir_model,access_ir_model,base.model_ir_model,event.group_event_user,1,0,0,0
```

Comment:
I don't understand this fix

'A event user need to be able to create/edit  mail template for the
event.registration.
So the read access on ir.model is needed to be able to edit the mail
template.'

What do you mean by that ? Either there is an issue with model when people create templates (which is something doable outside of event) and we have to try to sudo access to ir.model, either you wanted to highlight a specific flow ?